### PR TITLE
Css Task 4

### PR DESCRIPTION
**Pull request by dudzon**
https://dudzon.github.io/ui-internship-lwo2/css_tasks/task4

### CODE REVIEW

File: css_tasks/task4/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task 4</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <section class = "schedule">
```
Comment (tkachukivan) [inline css_tasks/task4/index.html]: you don't need spaces before and after  `=` in attributes

File: css_tasks/task4/index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      <h3 class="schedule__header">Weekly Schedule</h3>
+      <input type="checkbox" id="input--leisure" class="schedule__input--leisure" />
```
Comment (tkachukivan) [inline css_tasks/task4/index.html]: FYI: if your classes is bem modifiers you can't use them without base class, i meat it shoud be `class="schedule__input schedule__input--leisure"`

and this id-s should be written with one `-` as `input-leisure` they for your `label` tag and have not connection with your styles

File: css_tasks/task4/index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            <th>Tuesday</th>
+            <th>Wednesday</th>
+            <th>Thursday</th>
+            <th>Friday</th>
+            <th>Saturday</th>
+            <th>Sunday</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="table__leisure">
```
Comment (tkachukivan) [inline css_tasks/task4/index.html]: probably not table but row/table-row

File: css_tasks/task4/main.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+.schedule__header {
+  margin: 2rem;
+  text-align: center;
+}
+.schedule__input--leisure {
+  margin-left: 10rem;
+}
+.schedule__input--leisure,
+.schedule__input--sport,
+.schedule__input--knowledge {
```
Comment (tkachukivan) [inline css_tasks/task4/main.css]: looks like it is for general classes but not bem modifier

File: css_tasks/task4/main.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  height: 100%;
+  z-index: -20;
+}
+tbody tr th {
+  background-color: var(--tableRowHeaderColor);
+}
+tbody tr:hover {
+  background: #ffa;
+}
+tbody tr td:hover::after {
```
Comment (tkachukivan) [inline css_tasks/task4/main.css:93]: generally it is better to separate with new line every style block, it is a bit easier to read
```
tbody tr th {
  background-color: var(--tableRowHeaderColor);
}

tbody tr:hover {
  background: #ffa;
}
```

**Review by tkachukivan — CHANGES_REQUESTED** [review]
please fix filtering(comments more for future), when I filter by Leisure and Sport row/columns highlighting is broken

optional: you can make this checkboxes checked by default, and when user uncheck something not show this kind of activities, looks like better ui.

**Review by tsvetkovv — APPROVED** [review]

File: css_tasks/task4/index.html (hunk 1 above)
Comment (dudzon) [reply css_tasks/task4/index.html]: Done

File: css_tasks/task4/index.html (hunk 2 above)
Comment (dudzon) [reply css_tasks/task4/index.html]: Classes naming has been refactored

File: css_tasks/task4/index.html (hunk 3 above)
Comment (dudzon) [reply css_tasks/task4/index.html]: Done

**Pull request merged**